### ISSUE TITLE
Use IT_EMAIL_ADDRESS for internal server errors and update description

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#8372]
#### Description

We should update [internal server errors](https://github.com/civiform/civiform/blob/1a21f67dc4975e873423de4bb3e7f2381c61ed13/server/app/views/errors/InternalServerError.java#L56) to send emails to the IT email address

If an IT email address isn't provided, we should fall back to the support email.

Currently, IT email address is only used when [durable jobs fail](https://github.com/civiform/civiform/blob/1a21f67dc4975e873423de4bb3e7f2381c61ed13/server/app/durablejobs/AbstractDurableJobRunner.java#L59)

We should update the [setting description](https://github.com/civiform/civiform/blob/1a21f67dc4975e873423de4bb3e7f2381c61ed13/server/app/services/settings/SettingsManifest.java#L1707) to "This email address receives error notifications from CiviForm when there is an internal server error or a durable job fails."

#### What priority is this?

Low

#### Associated civic entity

This was something that came up when looking into an issue for Arkansas

#### Done when:

What conditions need to be met before this issue can be closed?

**Comment by NotAPlumber505** [comment]
Hi, I'm working with a group of 2 other CS students from Florida International University. We are part of the Tech Talent Academy program where we are required to join an open source project and make a contribution. We were wondering if we could tackle this issue

**Comment by melanie-exygy** [comment]
Hi @NotAPlumber505, here is some guidance for how to get started contributing to CiviForm:  [Technical contribution guide](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#outside-contributors-not-part-of-exygy-google-or-a-civic-entity).  You'll find an invite there for our [CiviForm Slack](https://join.slack.com/t/civiform/shared_invite/zt-niap7ys1-RAICICUpDJfjpizjyjBr7Q), which you're welcome to join.
If you decide this is the first issue that you want to work on, please confirm in another comment on this issue and I'll assign it to you.  Thanks for you interest in CiviForm!

**Comment by Lynnshelsie1** [comment]
hello, wondering if I can help solve this issue may I have access to this?

**Comment by melanie-exygy** [comment]
@dkatzz I'd like to make the issue requirements more specific so that @Lynnshelsie1 can work on this.  Do you have a preference for which of these two directions to go?  "either remove this setting or replace it with some instances of support_email_address".
Also, do you have a suggestion for better wording on the settings descriptions?  Both of those make sense to me, but I wonder if you'd prefer "This email address receives error notifications from CiviForm when a durable job fails."

**Comment by dkatzz** [comment]
> [@dkatzz](https://github.com/dkatzz) I'd like to make the issue requirements more specific so that [@Lynnshelsie1](https://github.com/Lynnshelsie1) can work on this. Do you have a preference for which of these two directions to go? "either remove this setting or replace it with some instances of support_email_address". Also, do you have a suggestion for better wording on the settings descriptions? Both of those make sense to me, but I wonder if you'd prefer "This email address receives error notifications from CiviForm when a durable job fails."

Thank you, @melanie-exygy ! I think we should have internal server errors go to the IT email address as well and also change the description to "This email address receives error notifications from CiviForm when there is an internal server error or a durable job fails." Does that sound good? If so, I can update the description.

**Comment by melanie-exygy** [comment]
> > [@dkatzz](https://github.com/dkatzz) I'd like to make the issue requirements more specific so that [@Lynnshelsie1](https://github.com/Lynnshelsie1) can work on this. Do you have a preference for which of these two directions to go? "either remove this setting or replace it with some instances of support_email_address". Also, do you have a suggestion for better wording on the settings descriptions? Both of those make sense to me, but I wonder if you'd prefer "This email address receives error notifications from CiviForm when a durable job fails."
> 
> Thank you, [@melanie-exygy](https://github.com/melanie-exygy) ! I think we should have internal server errors go to the IT email address as well and also change the description to "This email address receives error notifications from CiviForm when there is an internal server error or a durable job fails." Does that sound good? If so, I can update the description.

That sounds good to me.  It definitely makes more sense for internal server errors to send an email to IT rather than support.  Thanks!

**Comment by dkatzz** [comment]
> > > [@dkatzz](https://github.com/dkatzz) I'd like to make the issue requirements more specific so that [@Lynnshelsie1](https://github.com/Lynnshelsie1) can work on this. Do you have a preference for which of these two directions to go? "either remove this setting or replace it with some instances of support_email_address". Also, do you have a suggestion for better wording on the settings descriptions? Both of those make sense to me, but I wonder if you'd prefer "This email address receives error notifications from CiviForm when a durable job fails."
> > 
> > 
> > Thank you, [@melanie-exygy](https://github.com/melanie-exygy) ! I think we should have internal server errors go to the IT email address as well and also change the description to "This email address receives error notifications from CiviForm when there is an internal server error or a durable job fails." Does that sound good? If so, I can update the description.
> 
> That sounds good to me. It definitely makes more sense for internal server errors to send an email to IT rather than support. Thanks!

Great - updated the description, but feel free to make any edits. Thanks!

**Comment by melanie-exygy** [comment]
> hello, wondering if I can help solve this issue may I have access to this?

Hi @Lynnshelsie1, this is now ready for you to work on.  I've assigned the issue to you.  You can start by forking the repository.  This [Getting Started wiki page](https://github.com/civiform/civiform/wiki/Getting-started) will help you set up your environment.

**Comment by gwendolyngoetz** [comment]
`IT_EMAIL_ADDRESS` isn't a required field, it should fall back to `SUPPORT_EMAIL_ADDRESS` if not set. We used to do that somewhere, but I don't recall where or if it was removed.

**Comment by dkatzz** [comment]
> SUPPORT_EMAIL_ADDRESS

Good call! Added that note to the description - here is the support email address https://github.com/civiform/civiform/blob/00d7f41c3240776a2820bccb52142c3a09958f09/server/conf/application.conf#L108

**Comment by dkatzz** [comment]
@Lynnshelsie1 given its been a few months, I'm going to unassign you from this issue. If you're still working on it, feel free to reassign yourself.
